<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doctorRegister</class>
 <widget class="QDialog" name="doctorRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size:15px
}</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size:15px
}</string>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size:15px
}</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size:15px
}</string>
   </property>
   <property name="text">
    <string>Specialization:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size:15px
}</string>
   </property>
   <property name="text">
    <string>Your Assistant:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>0</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#0000ff;&quot;&gt;Please enter your details&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="assistantEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="errorExisting">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size:15px
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#aa0000;&quot;&gt;*Already Existing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size:15px
}</string>
   </property>
   <property name="text">
    <string>Note: Please Make sure that the ID starts with 01</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorCheck">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>670</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size:15px
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#aa0000;&quot;&gt;*All fields must be filled&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorID">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size:15px
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;ERROR: ID IS NOT STARTING WITH 01&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPassRe">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Re-type Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="repassEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPassError">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;*Password Does Not Match&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>209</x>
     <y>265</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Select Field</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cardiologist</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dermatologist</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>General Surgeon</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Neurologists</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pediatrician</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Psychiatrist</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Optometrist</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dentist</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Orthopedist</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dietitian</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelHours">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size:15px
}</string>
   </property>
   <property name="text">
    <string>Working Interval:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxInterval">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButtonAM">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>AM</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonPM">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PM</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelHoursDays">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>520</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size:15px
}</string>
   </property>
   <property name="text">
    <string>Working Days:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxDays">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>520</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="checkBoxU">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>U</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxT">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>T</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxM">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxW">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxR">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelHours_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
font-size:15px
}</string>
   </property>
   <property name="text">
    <string>Working Hours:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxHours">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="comboBoxFrom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select Field</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12:00</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelFrom">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>From:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select Field</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11:00</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11:30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12:00</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonRegister">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>658</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 900 9pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
